viewlib.php

```diff
@@ -257,8 +257,9 @@ function bigbluebuttonbn_view_render_recordings(&$bbbsession, $enabledfeatures,
     );
     if ($enabledfeatures['importrecordings']) {
         // Get recording links.
+        $bigbluebuttonbnid = $bbbsession['bigbluebuttonbn']->id;
         $recordingsimported = bigbluebuttonbn_get_recordings_imported_array(
-            $bbbsession['course']->id, $bigbluebuttonbnid, $enabledfeatures['showroom']
+            $bbbsession['course']->id, $bigbluebuttonbnid,  true
```

Comment:
I haven't seen the details but the actual fix would not be replacing `$enabledfeatures['showroom']` with `$enabledfeatures['showrecordings']` ?

Reply:
Basically this function is called  with $enabledfeatures['showrecording'] = true in any case (see bigbluebuttonbn_view_render), so in our case, this means that this would be equivalent to setting it to true.